openedx_tagging/core/tagging/rest_api/v1/permissions.py

```diff
@@ -1,7 +1,7 @@
 """
 Tagging permissions
 """
-
+import rules  # type: ignore
```

Comment:
It's better to use `# type: ignore[import]` so that when `rules` implements type hints, mypy will give us a warning that this `ignore` is no longer needed, and we can remove it to get the type hints working. If you just use `type: ignore` it will be ignored forever, even if `rules` implements type hints at some point.

Reply:
Updated here: https://github.com/openedx/openedx-learning/pull/78/commits/9a15562c8509eee795dff0a939a9310b8a33af3f